tests/virt_autotest/virt_v2v_dst.pm

```diff
@@ -24,8 +24,20 @@ sub get_script_run {
     handle_sp_in_settings_with_sp0("GUEST_LIST");
     my $guests = get_var("GUEST_LIST");
     our $virt_v2v_log;
-
-    my $pre_test_cmd = "/usr/share/qa/virtautolib/lib/virt_v2v_test.sh -s $src_ip -u $src_user -p $src_pass -i \"$guests\" 2>&1 | tee $virt_v2v_log";
+    # Pass guest registration code used for guest installation in dynamically via GUEST_SCC_REGCODE
+    # and GUEST_SCC_REGCODE_LTSS. If they are not provided, SCC_REGCODE and SCC_REGCODE_LTSS_15 will
+    # be used. If there are multiple guest patterns, their corresponding registration codes should
+    # also be specified in the same order and separated by comma symbols. Empty value is allowed. For
+    # example, regcode_guest1,regcode_guest2,,regcode_guest4, because regcode_guest3 is emety, empty
+    # value is passed in to preserve order. For multiple guest patterns, an empty registration code
+    # for specific guest pattern will not be filled out by any default value and at the same time this
+    # means registration code is not needed for it at all.
+    my $guest_regcode = get_var("GUEST_SCC_REGCODE", "");
+    my $guest_regcode_ltss = get_var("GUEST_SCC_REGCODE_LTSS", "");
+    $guest_regcode = join(',', (get_var("SCC_REGCODE", "")) x scalar(split(',', $guests))) if (!$guest_regcode);
+    $guest_regcode_ltss = join(',', (get_var("SCC_REGCODE_LTSS_15", "")) x scalar(split(',', $guests))) if (!$guest_regcode_ltss);
```

Comment:
How about putting above block into some lib function like init_guest_regcode?

Reply:
Is it really necessary ? I do not think this is very necessary. Not very general use case.

Reply:
I think necessary, since 4 files are repeating on this part...

Reply:
I really can not see a point to do this. Not very meaningful.

Reply:
It is not following "DRY" rule in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md. @okurz  HDYT?

Reply:
Done